File: Python/Program3.py
head=-1
size=int(input("Enter the size of the stack"))
stack=[]
while True:
    option=int(input("Enter 1 for push, 2 for pop, 3 for peek, 4 for isFull, 5 for isEmpty, and 0 to exit the program"))
    if(option==1):
        if(head==(size-1)):
            print("Stack is full")
        else:
            head+=1
            stack.append(int(input("Enter the element to push")))
    if(option==2):
        if(head==-1):
            print("Stack is Empty")
        else:
            print("The popped element is {}".format(stack[head]))
            head-=1
            stack.pop()
    if(option==3):
        if(head==-1):
            print("Stack is empty")
        else:
            print("The head element is {}".format(stack[head]))
    if(option==4):
        if(head==(size-1)):
            print("Stack is full")
        else:
            print("Stack is not full")
    if(option==5):
        if(head==-1):
            print("Stack is empty")
        else:
            print("Stack is not empty")
    if(option==0):
        break

File: C++/Program3.cpp
#include <iostream>
using namespace std;
int main(){
    int size,option,head=-1;
    cout<<"Please Enter the size of the stack: ";
    cin>>size;
     int st[size];
    do{
        cout<<"\nEnter 1 for Push, 2 for Pop, 3 for peek, 4 for isFull, 5 for isEmpty, and 0 to exit the program\n";
        cin>>option;
       
        switch(option){
            case 1:
            if(head==(size-1)){
                cout<<"\nStack is full";
                break;
            }
            else{
                int temp;
                cout<<"\nEnter the element to push";
                cin>>temp;
                head++;
                st[head]=temp;
                break;
            }
            case 2:
            if(head==-1){
                cout<<"\nStack is empty";
                break;
            }
            else{
                cout<<"The popped element is: "<<st[head];
                head--;
                break;
            }
            case 3:
            if(head==-1){
                cout<<"\nStack is empty";
                break;
            }
            else{
                
                cout<<"\nThe head element is: "<<st[head];
                break;
            }
            case 4:
            if(head==(size-1)){
                cout<<"\nStack is full";
                break;
            }
            else{
                cout<<"\nStack is not Full";
                break;
            }
            case 5:
            if(head==-1){
                cout<<"\nStack is empty";
                break;
            }
            else{
                cout<<"\nStack is not empty";
                break;
            }
            case 0:
            cout<<"\nExiting the program";
            break;

            default:
            cout<<"\nInvalid input. Exiting Program";
        }

    }while(option!=0);
    return 0;
}